faq/instances.mdx

```diff
@@ -257,4 +257,43 @@ Check the [Instances availability guide](/console/account/reference-content/prod
 
 `FR-PAR-2` is our sustainable and environmentally efficient Availability Zone (AZ) in Paris.
 
-This Region is entirely powered by renewable (hydraulic) energy. It also has an energetic footprint 30-40% lower than traditional data centers, thanks to the fact that it does not require air conditioning. Learn more about [our environmental commitment](https://www.scaleway.com/en/environmental-leadership/).
\ No newline at end of file
+This Region is entirely powered by renewable (hydraulic) energy. It also has an energetic footprint 30-40% lower than traditional data centers, thanks to the fact that it does not require air conditioning. Learn more about [our environmental commitment](https://www.scaleway.com/en/environmental-leadership/).
+
+## Network
+
+## Which Instances support IP Mobility?
+
+All ranges are compatible except for Instances using the “bootscript” feature, which has [reached its end of life (EOL)].
+
+### Can I enable “Routed IP” on my bootscript Instance?
+
+Instances using a bootscript are not compatible with Routed IPs. As the bootscript feature has reached its end of life (EOL), please refer to [this tutorial](/compute/instances/troubleshooting/bootscript-eol/) for migration instructions.
+
+### Can I revert the migration to “Routed IP”?
+
+No, the migration is permanent.
+
+### Will my Instance retain its public IPv4 after migration?
+
+Yes, your existing flexible public IPv4 will be converted to a routed one and will remain unchanged after migration.
+
+### Will my Instance retain its public IPv6 after migration?
+
+No, you will receive a new IPv6 after migration.
+
+### Will my Instance retain its private IPv4 after migration?
+
+No, the Scaleway private IP will be removed after the migration.
+
+### Will my Instance retain its VPC private IPv4 after migration?
+
+Yes, your VPC configuration will remain intact after migration to a routed public IPv4.
+
+### Can a resource with a routed IP communicate with a resource with a NAT IP?
+
+Yes, they can communicate with each other using their public IPs.
+
+### Are there any limitations on IP autoconfiguration with the routed IP feature?
+
+Currently, additional routed IPv6 addresses do not autoconfigure on CentOS 7, 8, 9, Alma 8, 9, Rocky 8, 9 after migration.
+Note that, both additional routed IPv4 and IPv6 addresses are not autoconfigured post-migration on Ubuntu 20.04 focal. However, the primary IPv6 continues to be configured via SLAAC. These limitations are currently being addressed.
```

Comment:
doubts on "Note that", maybe replace by "Also" ? I can't remember what is the rule with this formulation